pkg/tsdb/azuremonitor/metrics/azuremonitor-datasource.go

```diff
@@ -112,7 +112,7 @@ func (e *AzureMonitorDatasource) buildQueries(queries []backend.DataQuery, dsInf
 			azureURL = ub.BuildMetricsURL()
 			// POST requests are only supported at the subscription level
 			filterInBody = false
-			resourceUri := ub.buildResourceURI()
+			resourceUri, _ := ub.buildResourceURI()
```

Comment:
The error should be checked here so we can exit early and provide information to the user about the issue with the query